oak_client/src/attestation.rs

```diff
@@ -91,13 +110,19 @@ impl RemoteAttestationVerifier {
         }
     }
 
-    fn check_tee_report(&self, extension: &str) -> anyhow::Result<()> {
+    /// Check that the [`extension`] contains [`RemoteAttestationVerifier::tee_measurement`] and
+    /// a SHA-256 digest of the [`public_key`].
```

Comment:
Also it checks the measurement, right? Which I would expect to see here as a param too, or at least mentioned in the comment.